content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Poseidon-Firmament - An alternate scheduler
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+**Current release of Poseidon-Firmament scheduler is an <code> alpha </code> release.**
+ 
+Poseidon-Firmament scheduler is an alternate scheduler alongside the default Kubernetes scheduler. 
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+## Introduction	
+
+Poseidon is the integration glue for the  [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) with Kubernetes. Poseidon-Firmament scheduler augments the current Kubernetes scheduling capabilities. It incorporates novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. Firmament scheduler models workloads and clusters as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
```

Comment:
Other tweaks:
Poseidon is a set of services (what is it besides glue? Libraries????) that integrates the [Firmament scheduler](.....) with Kubernetes.

Reply:
Changed this line.